### DISCUSSION TITLE
I cant seem to get sorting-classes-in-function-calls to work

### DISCUSSION THREAD

**Discussion by kirkstrobeck** [tailwindlabs/tailwindcss#12446] [Help] [answered]
1. When I use the following code, `classNames` is not sorted
2. `prettier-plugin-tailwindcss` is working for the `className` attribute

https://github.com/tailwindlabs/prettier-plugin-tailwindcss#sorting-classes-in-function-calls

From my `.prettierrc`

```json
{
  ...,
  "tailwindFunctions": ["classNames"]
}
```

here’s code

```tsx
        <div
          className={classNames(
            'bg-blue-50',
            'font-medium',
            'inline-flex',
            'items-center',
          )}
        />
```

### Versions

- tailwindcss 3.3.3
- prettier 3.0.3
- prettier-plugin-tailwindcss 0.5.7

**Comment by makibat** [comment]
Did you add `"plugins": ["prettier-plugin-tailwindcss"]` to your .prettierrc? 

If that doesn't do the trick, I suggest trying older versions of prettier https://github.com/tailwindlabs/tailwindcss/discussions/10717#discussioncomment-6621440

**Comment by RobinMalfait** [comment, marked as answer]
Hey!

Each string argument in the function call will be sorted individually, but the arguments themselves won't be sorted.

```tsx
<div
  className={classNames(
    'bg-blue-50 font-medium inline-flex items-center', // These will be sorted
  )}
/>
```

Hope this helps!

**Reply by kirkstrobeck** [reply]
Oh! That addresses my issue, thank you!

What would be the best approach to add argument sorting as the default or optional behavior? 

Also, do you know if the docs mention this?
